solutions/kubernetes/functions.sh

```diff
@@ -218,115 +218,4 @@ function ssh_all_nodes() {
 
 function ssh_primary_node() {
     ssh -o 'StrictHostKeyChecking=no' "$PRIMARY_NODE" $*
-}
-
-function setup_awscli() {
-   add_secondary_separator "Setup awscli"
-   
-   # Configure plugin, api and endpoints.
-   add_common_separator "Setup aws s3 plugin endpoints"
-   aws configure set plugins.endpoint awscli_plugin_endpoint
-   check_status "Failed to set awscli s3 plugin endpoint"
-   add_common_separator "Setup aws s3 endpoint url"
-   aws configure set s3.endpoint_url $ENDPOINT_URL
-   check_status "Failed to set awscli s3 endpoint url"
-   add_common_separator "Setup default aws region"
-   aws configure set default.region us-east-1
-   check_status "Failed to set default aws region"
-   add_common_separator "Setup awscli s3api endpoint url"
-   aws configure set s3api.endpoint_url $ENDPOINT_URL
-   check_status "Failed to set awscli s3 api endpoint url"
-
-   # Setup awscli authentication.
-   add_common_separator "Setup aws access key"
-   aws configure set aws_access_key_id $ACCESS_KEY
-   check_status "Failed to set awscli access key"
-   add_common_separator "Setup aws secret key"
-   aws configure set aws_secret_access_key $SECRET_KEY
-   check_status "Failed to set awscli secret key"
-   cat /root/.aws/config
-   add_primary_separator "Successfully installed and configured awscli"
-}
-
-function install_awscli() {
-   add_primary_separator "\tInstall and setup awscli"
-
-   add_secondary_separator "Check and install pip3 if not present:"
-   if ! which pip3; then
-      yum install python3-pip
-   fi
-
-   add_secondary_separator "Installing awscli"
-   pip3 install awscli
-   pip3 install awscli-plugin-endpoint
-
-   if ! which aws; then
-      add_common_separator "AWS CLI installation failed"
-      exit 1
-   fi
-}
-
-# Install yq 4.13.3
-
-function install_yq() {
```

Comment:
Is this also defined somewhere else and hence needs to be removed?

Reply:
it is defined in cortx-deploy-functions.sh, Ok reverted functions.sh change.